src/_pytest/config/__init__.py

```diff
@@ -203,7 +206,10 @@ def get_plugin_manager():
     return get_config().pluginmanager
 
 
-def _prepareconfig(args=None, plugins=None):
+def _prepareconfig(
+    args: Optional[Union[List, Tuple]] = None,
```

Comment:
Looks like this can also be a `py.path.local`. Even though it's not a real type yet, if you recall we decided to add it anyway so once typing is added to `py.path` it will start working.

Reply:
Will change it to `args: Optional[Sequence[Union[str, py.path.local]]] = None,`.